okio/src/main/java/okio/RealBufferedSource.java

```diff
@@ -233,6 +233,16 @@ public RealBufferedSource(Source source) {
     return buffer.readLongLe();
   }
 
+  @Override public long readDecimalLong() throws IOException {
+    request(20);
```

Comment:
@swankjesse Is this the right way to do this? I have to request the maximum width that a decimal long could be.